src/services/WebhookService.ts

```diff
@@ -10,25 +10,57 @@ const fetchShopifyWebhooks = async (payload?:  any): Promise <any>  => {
   });
 }
 
-// TODO: add the service endpoint for the new order webhook, cancelled order webhook, payment status webhook, order return webhook & new product webhook.
-const webhookEndpointUrls = {
-  'NEW_ORDERS': '',
-  'CANCELLED_ORDERS': '',
-  'PAYMENT_STATUS': '',
-  'RETURNS': '',
-  'NEW_PRODUCTS': '',
-  'DELETE_PRODUCTS': 'service/subscribeProductDeleteWebhook',
-  'BULK_OPERATIONS_FINISH': 'service/subscribeFileStatusUpdateWebhook'
+const webhookParameters = {
+  'NEW_ORDERS': {
+    'topic': 'orders/create',
+    'endpoint': ''
+  },
+  'CANCELLED_ORDERS': {
+    'topic': 'orders/cancelled',
+    'endpoint': ''
+  },
+  'PAYMENT_STATUS': {
+    'topic': '',
+    'endpoint': ''
+  },
+  'RETURNS': {
+    'topic': '',
+    'endpoint': ''
+  },
+  'NEW_PRODUCTS': {
+    'topic': 'products/create',
+    'endpoint': ''
+  },
+  'DELETE_PRODUCTS': {
+    'topic': 'products/delete',
+    'endpoint': 'deleteProductFromShopify'
+  },
+  'BULK_OPERATIONS_FINISH': {
+    'topic': 'bulk_operations/finish',
+    'endpoint': 'uploadedFileStatusUpdateFromShopify'
+  },
+  'INVENTORY_LEVEL_UPDATE': {
+    'topic': 'inventory_levels/update',
+    'endpoint': 'inventoryLevelUpdateFromShopify'
+  },
+  'ORDER_PAID': {
+    'topic': 'orders/paid',
+    'endpoint': 'orderPaidNotificationFromShopify'
+  }
 } as any
 
 const subscribeWebhook = async (payload?: any, id?: string): Promise <any> => {
-  const endpointUrl = webhookEndpointUrls[id as string];
-  if(!endpointUrl) {
+  const topic = webhookParameters[id as string].topic;
```

Comment:
Make id in parameters that way we do not have to explicitly cast as string

Reply:
Done sir